data/splits/evaluation_tasks.txt

```diff
@@ -23,4 +23,7 @@ winogrande validation (grammar) additional_ph
 sandbox_audio_quality
 sandbox_figure_descriptions
 sandbox_lamecows
-sandbox_scambaiting
\ No newline at end of file
+sandbox_scambaiting
+Sentence Formality Annotation (extensive data cleaning)
+Explanation Acceptability (CommonsenseQA)
+Human evaluation - quals
```

Comment:
I suggest we rename this file to`data/splits/evaluation_tasks_easy.txt` and make sure that it has empty intersection with `data/splits/evaluation_tasks_hard.txt` (currently it has some overlap).

Reply:
it is mostly overlapped, hard is strictly a superset of easy with the addition of the hard tasks. is that not what we want?

Reply:
It would be nice to keep them disjoint. Basically, we would report these two evaluation sets to report two distinct numbers for each models (easy vs. hard).